tools/integration_tests/util/operations/file_operations.go

```diff
@@ -192,3 +193,41 @@ func ReadFileSequentially(filePath string, chunkSize int64) (content []byte, err
 	}
 	return
 }
+
+func WriteFileSequentially(filePath string, fileSize int64, chunkSize int64) (err error) {
+	file, err := os.OpenFile(filePath, os.O_RDWR|syscall.O_DIRECT|os.O_CREATE, FilePermission_0600)
+	if err != nil {
+		log.Printf("Error in opening file:%v", err)
+	}
+
+	// Closing file at the end.
+	defer CloseFile(file)
+
+	var offset int64 = 0
+
+	for offset < fileSize {
+		// Get random 200MB or remaining filesize data into chunk.
+		if (fileSize - offset) < chunkSize {
+			chunkSize = (fileSize - offset)
+		}
+		chunk := make([]byte, chunkSize)
+		_, err = rand.Read(chunk)
+		if err != nil {
+			log.Fatalf("error while generating random string: %s", err)
+		}
+
+		var numberOfBytes int
```

Comment:
Can't we do in the next line?
numberOfBytes, err **:=** file.Write(chunk)

Reply:
No it is giving error.
```
../util/operations/file_operations.go:224:4: result parameter err not in scope at return
	../util/operations/file_operations.go:222:18: inner declaration of var err error
```